sensor/common/scannerdefinitions/http_handler.go

```diff
@@ -0,0 +1,82 @@
+package scannerdefinitions
+
+import (
+	"io"
+	"net/http"
+	"net/url"
+
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/pkg/clientconn"
+	"github.com/stackrox/rox/pkg/httputil"
+	"github.com/stackrox/rox/pkg/mtls"
+	"github.com/stackrox/rox/pkg/set"
+	"github.com/stackrox/rox/pkg/utils"
+	"google.golang.org/grpc/codes"
+)
+
+var (
+	headersToProxy = set.NewFrozenStringSet("If-Modified-Since", "Accept-Encoding")
+)
+
+// scannerDefinitionsHandler handles requests to retrieve scanner definitions
+// from Central.
+type scannerDefinitionsHandler struct {
+	http.Handler
```

Comment:
I think putting this in here is best if you plan on doing something like 
`&scannerDefinitionsHandler{Handler: <something>}`. Implementing 
`func (h *scannerDefinitionsHandler) ServeHTTP(writer http.ResponseWriter, request *http.Request)` implicitly makes this an `http.Handler`, so I don't think this is needed. That being said, I like to add `var _ http.Handler = (*scannerDefinitionsHandler)(nil)` above struct definitions which are meant to implement interfaces. This ensures upon compilation that the struct does, in fact, implement the interfaces you expect. Also helps readability because we explicitly state which interfaces this struct is meant to implement